server/db/index.js

```diff
@@ -43,19 +43,103 @@ exports.getUsers = async (info) => {
   if (!usersinfo) {
     throw Error("Can't find any user");
   }
-  const users = usersinfo.map((user) => [user.username, user.profileImage]);
+  const users = usersinfo.map((user) => ({
+    username: user.username,
+    profileImage: user.profileImage,
+    friends: user.friends,
+  }));
   return users;
 };
 
 exports.addFriend = async (username, newfriend) => {
-  const friends = await db.Friend.findOneAndUpdate(
+  await db.User.findOneAndUpdate(
     { username },
     { $addToSet: { friends: newfriend } },
     { upsert: true }
   );
 };
 
-exports.getUserInfo = async (_id) => {
```

Comment:
Looks like getUserInfo is accidentally removed? Without this function, app brokes.